smartcontracts/sandbox/contracts/interfaces/IFleetManager.sol

```diff
@@ -4,7 +4,13 @@ pragma solidity ^0.8.19;
 /// @title Sandbox fleet manager
 interface IFleetManager {
     /// @notice check operator rights for project
-    /// @param _operator operator address
+    /// @param _node operator address
     /// @param _projectId project id
-    function isAllowed(address _operator, uint256 _projectId) external view returns (bool);
+    function isAllowed(address _node, uint256 _projectId) external view returns (bool);
```

Comment:
According current sprout design, the address that submit data to Router is operator address, not node address. Node address will introduce to sprout, but not design currently.